decoder/vaapidecoder_h264.cpp

```diff
@@ -1637,6 +1647,19 @@ YamiStatus VaapiDecoderH264::ensureContext(const SharedPtr<SPS>& sps)
             return status;
         // return YAMI_DECODE_FORMAT_CHANGE to info upper layer va context changed
         return YAMI_DECODE_FORMAT_CHANGE;
+    }else if(isResolutionChanged(sps)){
+        m_configBuffer.width = sps->frame_cropping_flag ? sps->m_cropRectWidth
```

Comment:
Not related to this commit, but there are so many place set or not set the video info and m_configBuffer... we'd better refact code to a better form